server/bin/pbench-dispatch.sh

```diff
@@ -88,11 +88,11 @@ if [[ ! -d "${receive_dir}" ]]; then
     exit 2
 fi
 
-quarantine=${qdir}/md5-${version}
-mkdir -p ${quarantine}
+bad_md5=${qdir}/md5-${version}
```

Comment:
Thank you for renaming this -- it was bothering me having a variable with the same name as a "command".

Reply:
I don't know why I missed that before!